cypress-tests/cypress/e2e/ConnectorUtils/Adyen.js

```diff
@@ -22,15 +22,19 @@ export const connectorDetails = {
         "paymentSuccessfulStatus": "requires_customer_action",
         "paymentSyncStatus": "succeeded",
         "refundStatus": "pending",
-        "refundSyncStatus": "pending"
+        "refundSyncStatus": "pending",
+        "customer_acceptance":null,
+        "setup_future_usage": "on_session",
     },
     "No3DS": {
         "card": successfulNo3DSCardDetails,
         "currency": "USD",
         "paymentSuccessfulStatus": "succeeded",
         "paymentSyncStatus": "succeeded",
         "refundStatus": "pending",
-        "refundSyncStatus": "pending"
+        "refundSyncStatus": "pending",
+        "customer_acceptance":null,
```

Comment:
Can we run a formatter on these JS files, maybe Prettier or something similar?

Can be done in a different PR, not a problem.